src/browser/modules/Stream/CypherFrame/PlanView.jsx

```diff
@@ -49,6 +49,8 @@ export class PlanView extends Component {
     this.ensureToggleExpand(props)
   }
   shouldComponentUpdate (props, state) {
+    this.props.assignVisElement && this.props.assignVisElement(this.el, this.plan)
```

Comment:
I don't think this is the lifecycle method to use for this. `componentDidUpdate` or `componentWillReceiveProps` would be more appropriate imo.